www/index.html

```diff
@@ -15,12 +15,12 @@
             <h2 class="mtbs">Sign up</h2>
             <label
               class="form-label form-label-small"
-              for="username"
-            >Username</label>
+              for="Email"
+            >Email</label>
             <input
               class="form-input form-input-small mbm"
-              name="username"
-              id="username"
+              name="email"
+              id="email"
```

Comment:
for browser that don’t support type=email, it will fallback to type=text, so it’s a great progressive enhancement